### PR TITLE
Update doc for CA1721 to mention exclusion of obsolete members

### LINKED ISSUE #4991: CA1721 no longer report on obsolete members
With the release of fxcop 3.x, the analyzer will stop reporting a diagnostic on members marked as `Obsolete`.

See https://github.com/dotnet/roslyn-analyzers/pull/3327

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6a95064e-a79d-0fcf-4409-a7a6a602a1cb
* Version Independent ID: c4409ae4-2725-2862-5ab3-bacc207ad3fe
* Content: [CA1721: Property names should not match get methods - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1721?view=vs-2019)
* Content Source: [docs/code-quality/ca1721.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/code-quality/ca1721.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-code-analysis**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

### PR DESCRIPTION
**Pull request by Evangelink**
Fix #4991

### CODE REVIEW

**Comment by PRMerger17** [comment]
@Evangelink : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/code-quality/ca1721.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ ms.workload:
 
 ## Cause
 
-The name of a member starts with 'Get' and otherwise matches the name of a property. For example, a type that contains a method that is named 'GetColor' and a property that is named 'Color' cause a rule violation.
+The name of a member starts with 'Get' and otherwise matches the name of a property and none of the 2 members are marked as `[Obsolete]`. For example, a type that contains a method that is named 'GetColor' and a property that is named 'Color' cause a rule violation.
```
Comment (mavasani) [inline docs/code-quality/ca1721.md:32]: I would just add new sentence at the end of existing one `This rule will not fire if either the property or the method is marked as [Obsolete].`

**Comment by Mikejo5000** [comment]
@Evangelink Please make the change noted. Thanks!

**Comment by PRMerger16** [comment]
@Evangelink : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
#sign-off

**Pull request merged**